hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileCreateHandlerTest.java

```diff
@@ -209,7 +210,7 @@ void handleWorksAsExpected() {
         assertEquals(1_234_567L, actualFile.expirationTime());
         assertEquals(contentsBytes, actualFile.contents());
         assertEquals(fileEntityNum.longValue(), actualFile.fileNumber());
-        assertEquals(false, actualFile.deleted());
+        assertFalse(actualFile.deleted());
         assertEquals(1_234L, recordBuilder.getCreatedFile());
         assertTrue(fileStore.get(1234L).isPresent());
     }
```

Comment:
The TODO just below this line needs more information. Ideally each TODO should cite an issue number that more fully describes what needs to be done.

Reply:
removed it